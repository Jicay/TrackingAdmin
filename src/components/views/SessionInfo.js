import React, {Component} from "react";
import {Link} from "react-router-dom";

export default class SessionInfo extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        const {id, date} = this.props;
        const url = '/sessions/' + id;
        return (
            <div className="row">
                <div className="col">
                    {id}
                </div>
                <div className="col">
                    {date}
                </div>
                <div className="col">
                    <Link to={url}>Voir l'Ã©tat de la session</Link>
                </div>
            </div>
        );
    }
}