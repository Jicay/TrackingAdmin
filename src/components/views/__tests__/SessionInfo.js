import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from "react-router";
import SessionInfo from "../SessionInfo";

it('renders correctly', () => {
    const tree = renderer
        .create(<MemoryRouter><SessionInfo id={10} date='2019-10-03 12:35:45'/></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});