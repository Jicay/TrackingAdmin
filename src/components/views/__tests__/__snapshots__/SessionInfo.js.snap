// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col"
  >
    10
  </div>
  <div
    className="col"
  >
    2019-10-03 12:35:45
  </div>
  <div
    className="col"
  >
    <a
      href="/sessions/10"
      onClick={[Function]}
    >
      Voir l'Ã©tat de la session
    </a>
  </div>
</div>
`;
